src/views/Gamefi/components/Cards/Farm.tsx

```diff
@@ -131,7 +127,7 @@ const FarmCard2: React.FC<FarmCard2Props> = ({
             </Flex>
             <Flex style={{ justifyContent: 'end', alignItems: 'end' }}>
               <div style={{ textAlign: 'end' }}>
-                <Text color="white">APR {apr}</Text>
+                <Text color="white"> {apr ? `APR: ${apr}` : ''}</Text>
```

Comment:
Here naman, 

`{apr && <Text>APR: ${apr}</Text> }`

Note: Not exactly pero para maiwasan yung pag render parin ng component ng walang laman. tho might affect yung structure since nawala component.